tests/test_config_flow.py

```diff
@@ -0,0 +1,163 @@
+"""Test the Somfy TaHoma config flow."""
+from asynctest import patch
+from custom_components.tahoma import config_flow
+from pyhoma.exceptions import BadCredentialsException, TooManyRequestsException
+
+from homeassistant import config_entries, setup
+
+from .common import MockConfigEntry
+
+
+async def test_form(hass):
+    """Test we get the form."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
+    
+    result = await hass.config_entries.flow.async_init(
+        config_flow.DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+
+    assert result["type"] == "form"
+    assert result["errors"] == {}
+
+    with patch("pyhoma.client.TahomaClient.login", return_value=True), patch(
+        "custom_components.tahoma.async_setup", return_value=True
```

Comment:
We have to mock this because we don't have a true hass instance?

Reply:
Yes, this was part of the test boilerplate in core when running hassfest. And it is also how most components (that I checked) handle this.